apk-generator/v2/app/templates/index.html

```diff
@@ -69,7 +69,17 @@
                     	<input type="radio" name="build-type" value="fdroid" required/> FDroid
                 	</label>
             	</li>	
-	</ul>	
+	</ul>
+
+        <div class="col-md-offset-3 col-xs-offset-1 col-xs-10 col-md-6" style="margin-top: -5px;">
+   	        <center>
+                    <div id="status" style="margin-bottom:10px;"></div>
+                    <a id="log-message" style="cursor:pointer;"> -Logs- </a>
+		    <pre id="build-log" style="display:none;">
```

Comment:
Put `</pre>` on the same line. 
Like `<pre id="build-log" style="display:none;"></pre>` ... Because, in a `pre` tag, all space matters and will be visible.